### PR TITLE
Add tool to visualize LR schedule

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

before run a task, know what the learning rate would change along the task.
add a tool to visualize LR schedule

## Modification

mv function tools.analysis_tools.analyze_logs.load_json_logs to mmcls.utils.logger
add a tool to visualize LR schedule

## BC-breaking (Optional)

No

## Use cases (Optional)

```python tools/visualizations/vis_lr.py configs/resnet/resnet50_b16x8_cifar100.py```

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/tools/visualization.md [hunk 1]
```diff
@@ -76,6 +76,49 @@ python ./tools/visualizations/vis_pipeline.py configs/resnet/resnet50_b16x8_cifa
...
+**Examples**：
+
+```bash
+python tools/visualizations/vis_lr.py configs/resnet/resnet50_b16x8_cifar100.py
+```
+
+When using Imganet, directly specify the size of Imganet, as below:
+
+```bash
+python tools/visualizations/vis_lr.py configs/swin_transformer/swin_base_384_evalonly_imagenet.py --dataset-size 1100000 --save-path ./swin_lr_polt.jpg
```
Comment (mzr1996) [inline docs/tools/visualization.md:119]:
```suggestion
python tools/visualizations/vis_lr.py configs/swin_transformer/swin_tiny_224_b16x64_300e_imagenet.py --dataset-size 1281167 --ngpus 8 --save-path ./swin_lr_polt.jpg
```

File: docs/tools/visualization.md [hunk 2]
```diff
@@ -76,6 +76,49 @@ python ./tools/visualizations/vis_pipeline.py configs/resnet/resnet50_b16x8_cifa
...
+Loading annotations maybe consume much time, you can directly specify the size of the dataset with `dataset-size` to save time.
+```
+
+**Examples**：
+
+```bash
+python tools/visualizations/vis_lr.py configs/resnet/resnet50_b16x8_cifar100.py
+```
+
+When using Imganet, directly specify the size of Imganet, as below:
```
Comment (mzr1996) [inline docs/tools/visualization.md:116]: ImageNet

File: docs_zh-CN/tools/visualization.md [hunk 3]
```diff
@@ -77,6 +77,51 @@ python ./tools/visualizations/vis_pipeline.py configs/resnet/resnet50_b16x8_cifa
...
+
+```
+
+**示例**：
+
+```bash
+python tools/visualizations/vis_lr.py configs/resnet/resnet50_b16x8_cifar100.py
+```
+
+当数据集为 Imganet 时，通过直接指定数据集大小来节约时间，并保存图片：
```
Comment (mzr1996) [inline docs_zh-CN/tools/visualization.md:119]: ImageNet

File: docs_zh-CN/tools/visualization.md [hunk 4]
```diff
@@ -77,6 +77,51 @@ python ./tools/visualizations/vis_pipeline.py configs/resnet/resnet50_b16x8_cifa
...
+
+```bash
+python tools/visualizations/vis_lr.py configs/swin_transformer/swin_base_384_evalonly_imagenet.py --dataset-size 1100000 --save-path ./swin_lr_polt.jpg
```
Comment (mzr1996) [inline docs_zh-CN/tools/visualization.md:122]: Same

**Review by mzr1996 — CHANGES_REQUESTED** [review]

File: docs/tools/visualization.md (hunk 2 above)
Comment (Ezra-Yu) [reply docs/tools/visualization.md]: fixed

**Review by mzr1996 — APPROVED** [review]
LGTM

**Pull request merged**